Common/cpp/Tools/UIRuntimeDecorator.cpp

```diff
@@ -59,6 +60,11 @@ void UIRuntimeDecorator::decorate(
   jsi_utils::installJsiFunction(uiRuntime, "_setGestureState", setGestureState);
   jsi_utils::installJsiFunction(
       uiRuntime, "_maybeFlushUIUpdatesQueue", maybeFlushUIUpdatesQueue);
+
+  jsi_utils::installJsiFunction(
+      uiRuntime, "_obtainPropFabric", obtainPropFunction);
+  jsi_utils::installJsiFunction(
+      uiRuntime, "_obtainPropPaper", obtainPropFunction);
```

Comment:
Can we move these lines into `#ifdef RCT_NEW_ARCH_ENABLED` above, next to `_updatePropsPaper` and `_measurePaper`?